src/eval-frame/components/reps/data-table-rep.jsx

```diff
@@ -1,17 +1,14 @@
 import React from "react";
 import ReactTable from "react-table";
 import PropTypes from "prop-types";
-import { get } from "lodash";
+
 import styled from "react-emotion";
 
 import "./react-table-styles.css";
 
-import { serializeForValueSummary } from "./rep-utils/value-summary-serializer";
-
 import ExpandableRep from "./rep-tree";
 
-import { getChildSummary } from "./rep-utils/get-child-summaries";
-import { RangeDescriptor } from "./rep-utils/rep-serialization-core-types";
+import { requestRepInfo } from "./request-rep-info";
```

Comment:
all the changes in this file are about using `requestRepInfo` rather than direct functions for serializing object descriptors. this is the key stepping stone that will eventually allow truly remote reps across iframes.